libbeat/processors/add_locale/add_locale.go

```diff
@@ -0,0 +1,47 @@
+package actions
+
+import (
+	"time"
+
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/processors"
+	"github.com/pkg/errors"
+)
+
+type addLocale struct {
+	timezone string
+}
+
+func init() {
+	processors.RegisterPlugin("add_locale", newAddLocale)
+}
+
+func newAddLocale(c common.Config) (processors.Processor, error) {
+	config := struct {
+		TimeZone string `config:"timezone"`
```

Comment:
Ok. Changed up to `time.Now().Zone()`. I was not sure if `Now()` always returns local time.